apps/wpcom-block-editor/src/wpcom/features/live-preview.tsx

```diff
@@ -0,0 +1,103 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+import domReady from '@wordpress/dom-ready';
+import { __, sprintf } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import { getQueryArg } from '@wordpress/url';
+import { useEffect } from 'react';
+
+/**
+ * Return true if the user is currently previewing a theme.
+ * FIXME: This is copied from Gutenberg, and that should be accessed from the `core/edit-site` store.
+ * @see https://github.com/WordPress/gutenberg/blob/053c8f733c85d80c891fa308b071b9a18e5194e9/packages/edit-site/src/utils/is-previewing-theme.js#L6
+ * @returns {boolean} isPreviewingTheme
+ */
+function isPreviewingTheme() {
```

Comment:
I don't understand, why can't we use the selector directly? Because in line 57 we are already doing

```
const { getSettings } = unlock( select( 'core/edit-site' ) );
```

Why can't we do the same for `isPreviewingTheme()`?

Reply:
Because there is no selector now, it's just a util function! We should create a selector that wraps `isPreviewingTheme` on the GB side.

Reply:
Ah.... you're right 🤦 🙏